lib/import/rummager_data_presenter.rb

```diff
@@ -7,6 +7,7 @@ def self.present_content(batch_data)
           item['content_id'],
           item['format'],
           item['index'],
+          item['is_withdrawn'] == 'true' ? 'withdrawn' : 'not_withdrawn',
```

Comment:
does this really come back as a string? `'true'`?

Also - if this field is meant to be treated like an enum, can we make the valid values constants somewhere? I'm not a fan of using string literals in this way as it's easy to have typos and not notice.